libs/community/langchain_community/cache.py

```diff
@@ -272,6 +273,18 @@ def lookup(self, prompt: str, llm_string: str) -> Optional[RETURN_VAL_TYPE]:
                     )
                     # In a previous life we stored the raw text directly
                     # in the table, so assume it's in that format.
+                    if "('type', 'chat')" in llm_string:
```

Comment:
Is it possible to check whether the data looks like a ChatGeneration and if so return it otherwise return Generation?

I'd like to avoid the code checking the contents of llm_string to reason about what model is being used